app/scripts/proactive/templates/job-template.html

```diff
@@ -21,7 +21,7 @@
     <% if (job["Generic Info"] && job["Generic Info"].length>0) { %>
     <genericInformation>
     <% _.each(job["Generic Info"], function(gi) { %>
-            <info name="<%=gi["Property Name"]%>" value="<%=gi["Property Value"]%>"/>
+            <info name="<%=gi["Property Name"]%>" value="<%= _.escape(gi["Property Value"]) %> "/>

```

Comment:
I think this an acceptable solution however we should be aware that if we want to validate the gi values as we do for variables, we will have to find another solution.